From ca01fdf2fee140a5c6da49222609ddd178748681 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:20 -0400
Subject: [ethernet] i40evf: clean up init error messages

Message-id: <1409663591-32531-38-git-send-email-sassmann@redhat.com>
Patchwork-id: 90513
O-Subject: [RHEL7.1 PATCH 037/148] i40evf: clean up init error messages
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

Add an error message when the admin queue message never completes, and
fix formatting on another one that was unnecessarily wrapped.

Change-ID: I8b8a4eb7629d741f09357250144023cd4a72231f
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 10bdd67b4a19e0e4cd96ca0591adcde80907874d)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 4d547ab..d3eafa3 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -1968,8 +1968,7 @@ static void i40evf_init_task(struct work_struct *work)
   }
   err = i40evf_send_api_ver(adapter);
   if (err) {
-   dev_err(&pdev->dev, "Unable to send to PF (%d)\n",
-    err);
+   dev_err(&pdev->dev, "Unable to send to PF (%d)\n", err);
    i40evf_shutdown_adminq(hw);
    goto err;
   }
@@ -1977,8 +1976,10 @@ static void i40evf_init_task(struct work_struct *work)
   goto restart;
   break;
  case __I40EVF_INIT_VERSION_CHECK:
-  if (!i40evf_asq_done(hw))
+  if (!i40evf_asq_done(hw)) {
+   dev_err(&pdev->dev, "Admin queue command never completed.\n");
    goto err;
+  }
 
   /* aq msg sent, awaiting reply */
   err = i40evf_verify_api_ver(adapter);
-- 
1.7.1